live-ingester/src/test/java/org/alfresco/hxi_connector/live_ingester/adapters/storage/connector/HttpFileUploaderIntegrationTest.java

```diff
@@ -84,14 +83,15 @@ static void beforeAll() throws IOException, InterruptedException
     }
 
     @Test
-    void testUpload() throws IOException
+    void testUpload()
     {
         // given
         List<String> initialBucketContent = s3StorageMock.listBucketContent(BUCKET_NAME);
-        @Cleanup
-        InputStream fileInputStream = new ByteArrayInputStream(OBJECT_CONTENT.getBytes());
+
+        File fileToUpload = new File(new ByteArrayInputStream(OBJECT_CONTENT.getBytes()));
```

Comment:
You are still constructing here new `InputStream` so maybe you should leave the `@Cleanup` annotation? 🤔

Reply:
fixed, but idk if we need it

Reply:
guess we need it :)